src/service/note-service.js

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+const getNotes = async (dispatch) => {
+  const token = JSON.parse(localStorage.getItem("token"));
+  const Headers = { authorization: token };
+  try {
+    const { data :{notes}, status } = await axios.get('/api/notes', { headers: Headers });
+    if(status>=200 && status<=300){
```

Comment:
You don't need to check the status if you use Axios. You can directly use catch block to handle calls that were not successful (>=200 && < 300)

Reply:
But what if we want to show specific error to user, if someone is unauthorized to particular content then we need to filter 401 status and if there's internal server error then 500. 
I've not implemented checking for rest of the status code here but may add later. In that case is it fine to use?